app/assets/javascripts/discourse/app/components/directory-table.js

```diff
@@ -0,0 +1,19 @@
+import Component from "@ember/component";
+
+export default Component.extend({
+  classNames: ["directory-table-container"],
+
+  setActiveHeader(id) {
```

Comment:
I like the pattern of the child calling up to this action, but I think `id` is inefficient. In the child you could pass `this.element`, then you wouldn't need to `getElementById`. You might not need an `id` at all in the child.

Reply:
I swear `this.element` was null, thats why I did this mess. I tried it again and now I can access it without directly accessing the dom. Fantastic!